From 9a66b0a1781c090ec4f7de44e1fe340ee9328010 Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Tue, 23 Jun 2015 16:52:17 +0800
Subject: [PATCH] HDMI:rk3128/rk3036: fix compile error introduced by
 pre-commit.

	hdcp_power_off_cb() was changed by commit
	90e99a4c359e2609512feb7433670b10815593b4.

Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 .../video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1_hdcp.c  | 2 +-
 .../video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1_hw.c    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1_hdcp.c b/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1_hdcp.c
index 0617e4f2c91f..0b067c0114c3 100644
--- a/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1_hdcp.c
+++ b/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1_hdcp.c
@@ -620,7 +620,7 @@ static int hdcp_power_on_cb(void)
  * Function: hdcp_power_off_cb
  *-----------------------------------------------------------------------------
  */
-static void hdcp_power_off_cb(void)
+static void hdcp_power_off_cb(struct hdmi *hdmi)
 {
 	unsigned int time;
 
diff --git a/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1_hw.c b/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1_hw.c
index 37351e193833..47ebf02d48f1 100644
--- a/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1_hw.c
+++ b/drivers/video/rockchip/hdmi/rockchip-hdmiv1/rockchip_hdmiv1_hw.c
@@ -854,7 +854,7 @@ int rockchip_hdmiv1_removed(struct hdmi *hdmi_drv)
 {
 	dev_info(hdmi_drv->dev, "Removed.\n");
 	if (hdmi_drv->ops->hdcp_power_off_cb)
-		hdmi_drv->ops->hdcp_power_off_cb();
+		hdmi_drv->ops->hdcp_power_off_cb(hdmi_drv);
 
 	rockchip_hdmiv1_control_output(hdmi_drv, -1);
 	rockchip_hdmiv1_set_pwr_mode(hdmi_drv, LOWER_PWR);
-- 
2.35.3

